info.rkt

```diff
@@ -1,6 +1,10 @@
 #lang info
 (define name "fpbench")
 (define version "2.0.3")
+
+(define racket-launcher-names '("fpbench"))
+(define racket-launcher-libraries '("main.rkt"))
```

Comment:
I'm think it should be `fpbench.rkt`

Reply:
Does racket-launcher-libraries not refer to the library file?

Reply:
I think it refers to the list of files that get run when the launcher is invoked

Reply:
Yep fixed